tensorflow/core/platform/s3/s3_file_system.cc

```diff
@@ -165,6 +165,20 @@ Status ParseS3Path(const string& fname, bool empty_object_ok, string* bucket,
   return Status::OK();
 }
 
+Status CheckForbiddenError(const Aws::Client::AWSError<Aws::S3::S3Errors>& error) {
```

Comment:
Let's make these functions `static` as they should not be visible to the outside of this translation unit.

Reply:
Done. 
But for my understanding, could you expand on why this is needed. Since these functions are only in this file, can they still be visible externally?

Reply:
They can be visible externally. For example, `nm` on Linux will list them. So, someone could dynamically load the shared object that results from compiling this and then call them directly.